docs/pages/docs/features/pipelines.mdx

```diff
@@ -39,15 +39,16 @@ An example Pipeline configuration in `turbo.json`:
       "dependsOn": ["^build"]
     },
     "test": {
-      "dependsOn": ["^build"],
+      "dependsOn": ["build"],
+      "outputs": [""],
+      "inputs": ["**/*.tsx", "**/*.ts"]
```

Comment:
```suggestion
      "inputs": ["test/**/*.test.tsx", "test/**/*.test.ts"]
```

Reply:
Let's restrict to a folder so that we don't scan `node_modules` unless we need to

Reply:
Used `src` and `test`